### ISSUE TITLE
Interpretation of Nomenclatural Status in CLB

### ISSUE THREAD

**Issue by yroskov** [CatalogueOfLife/backend#1181]
Something wrong with interpretation of Nomenclatural Status in the CoL. For example, Allocapnia curiosa Frison, 1942 (as thousands other valid names) is interpreted as "_potentially valid_" whereas expected to be "_valid_" (https://www.checklistbank.org/catalogue/3/dataset/1065/taxon/89137).

![image](https://user-images.githubusercontent.com/28574252/201777076-7e33bcfc-1975-4746-bed3-024056c8c02c.png)

Source data as they exported from TaxonWorks:

![image](https://user-images.githubusercontent.com/28574252/201777478-9500722f-3f21-41b8-a311-9a8b7a6a3594.png)

**Comment by yroskov** [comment]
Comment from @gdower: CLB should translate NOMEN_0000224 to NULL.

**Comment by mdoering** [comment]
NOMEN 224 stands for `ICZN valid`. "potentially valid" is the zoological label of `ACCEPTABLE` which is the "best" nomenclatural status entry in our enumeration and equal to legitimate names in botany:

```
  /**
   * Botany: Names that are validly published and legitimate
   * Zoology: Available name and potentially valid, i.e. not otherwise invalid
   * for any other objective reason, such as being a junior homonym.
   */
  ACCEPTABLE(ESTABLISHED, "nomen legitimum", null, "potentially valid"),
```

The problem with the zoological valid name status is that this depends on the subjective, taxonomic opinion.
A valid name can only be chosen in the light of the genus placement. Hence we only mark these names as potentially valid:

https://en.wikipedia.org/wiki/Valid_name_(zoology)#Subjectively_invalid_names

I agree this is unfortunate. But I fail to see how we can treat this better when separating names and taxa.
I sometimes feel this whole distinction just causes problems.

I guess we could add a higher valid entry and apply that to all potentially valid names which are also accepted, i.e. are linked to a Taxon not Synonym. Translating `NOMEN_0000224` to NULL is just hiding problems.

**Comment by yroskov** [comment]
Thank you, @mdoering.

@proceps, could you please have a look on the issue. What is your opinion as IZN commissioner and TW developer?

**Comment by proceps** [comment]
I do not know where the status comes from. TW does not have any statuses on _Allocapnia curiosa_. So this should be introduced during migration to CoL. @gdower should be able to comment on this. Anyway, it looks like `NOMEN_0000224` is properly assigned to the name. The question is, I do not understand why "Valid" (Nomen_0000224) in TW is translated to "potentially valid" in CoL? "Valid" in zoology is an equivalent of "Correct name" in botany. "Available" in zoology is an equivalent of "Validly published name" in botany. May be the difference of the use of term 'valid' in zoology and botany is the source of confusion.
 
In TW, valid status is calculated:
1. The name is valid by default.
2. The name is invalid if it has synonym relationship
3. The name is unavailable if it has unavailable nomenclatural status (nomen nudum, e.g.)
4. If the name has a synonym status, but the name is reinstated as valid, we can specifically assign "valid" status to the name, and this will overwrite the synonym (synonyms stays in the DB for historical reference).

**Comment by gdower** [comment]
The potentially valid status comes from the ChecklistBank importer which is ChecklistBank's interpretation of NOMEN to a [simplified/unified status](http://api.checklistbank.org/vocab/nomstatus). I suggested changing the interpretation of `NOMEN_0000224` to `NULL` because it doesn't quite match any of the ChecklistBank [simplified nomenclatural statuses](http://api.checklistbank.org/vocab/nomstatus) and it is confusing taxonomic experts. In my opinion, the ChecklistBank importer should just take the verbatim NOMEN statuses (e.g., [`ICZN valid` for `NOMEN_0000224`](https://ontobee.org/ontology/NOMEN?iri=http://purl.obolibrary.org/obo/NOMEN_0000224)) as the name statuses because they are higher resolution data than the simplified statuses of ChecklistBank.

What is the purpose of having these simplified name statuses? Who are the simplified name statuses intended for? The general public probably does not need nomenclatural statuses and experts probably want the actual nomenclatural statuses, so it seems like the simplified statuses might not meet the needs of most of our users. If unified name statuses are needed for building the backbone, then could there be a separate interpreted column that attempts to unify statuses and a more detailed column that gives the actual nomenclatural status? That might also provide a way to improve the interpretations over time based on feedback from our community. We could also meet the needs of nomenclators by providing the full nomenclatural statuses. From my perspective, it's always difficult trying to map name statuses into 9 simplified categories that don't quite semantically fit the data. I prefer working with NOMEN because I can just set the name status to what the name status is actually supposed to be, but the NOMEN statuses are being simplified by ChecklistBank to the point of confusing taxonomic experts.

**Comment by mdoering** [comment]
This needs a longer conversation, but I agree it is confusing as it is and I never was very happy with the current state. 

On the other hand I still like the goal of achieving a unified broad view on the status of a name that works across codes. Some are rather alike (available + validly published). But valid + correct are terms that only apply to a name in the context of a given taxonomic opinion (i.e. accepted/synonym). Would IPNI say a name is correct or ZooBank it is valid? They can only assert they are potentially valid, i.e. all objective nomenclatural rules have passed. But since we mostly use names as part of name usages I suppose we could just add a new entry for valid/correct names and use them if the usage is an accepted taxon. Or indeed switch to a much more fine grained vocabulary like NOMEN that separates between codes.

**Comment by mdoering** [comment]
I agree. If we would not have separated names from taxa in the data model this would not have happened.
Potentially valid is at least in use: https://en.wikipedia.org/wiki/Valid_name_(zoology)#Subjectively_invalid_names

For now I would simply add a new value for valid/correct names and make sure we use that for all accepted taxa. Although important it has great potential for breaking things, so I will keep that for 2023 I am afraid.

**Comment by mdoering** [comment]
As an interims solution we might simple change the label "potentially valid" to "valid"?

**Comment by yroskov** [comment]
> we might simple change the label "potentially valid" to "valid"?

It will be correct status for **accepted** names in the CoL.

**Comment by mdoering** [comment]
This is still an open and important issue.
Reading the comments again I am not sure the main problem is clear. We have a separate status for a nomenclatural (binomial) name (**nomStatus**) and the taxonomic use of that name (**taxStatus**). That did also exist in the previous COL ACEF model (`GSDNameStatus` & `Sp2000NameStatus`) and is also in DwC (`dwc:nomenclaturalStatus` & `dwc:taxonomicStatus`).

Calling a name _valid_ in the zoological sense only makes sense as a taxonomic status. It is not a nomenclatural value. So applying that to the name status in ColDP cannot yield back valid, but instead the interpreter picks the next best status from nomenclature. Here [botany and zoology differ](https://archive.bgbm.org/IAPT/biocode/biocode.html#Table):

<img width="569" alt="image" src="https://github.com/CatalogueOfLife/backend/assets/327505/0f65c479-25bb-40dc-854e-d6aaa20c9c98">

In botanical nomenclature on top of the availability ("validly published") there is also the idea of an [illegitimate name](https://en.wikipedia.org/wiki/Nomen_illegitimum) which includes homonyms. Fully determining a homonym requires the genus placement, so it is taking up some taxonomic decision. As the botanical code makes genus placements / combinations a nomenclatural governed act this is a proper nomenclatural name status. For zoology secondary homonyms depend on the taxonomy and thus are not an objective, universal name status. Nevertheless there are also nomenclatural rules on top of availability in zoology that can be checked universally for the name, regardless its taxonomy, e.g. [primary homonyms, objective synonyms, supressed names](https://en.wikipedia.org/wiki/Valid_name_(zoology)#Objectively_invalid_names). Passing these additional zoological rules corresponds more or less with the legitimate status in botany and some people have called this "potentially valid" in zoology. It is between available and valid, but still a purely objective, nomenclatural status. 

I don't know what the best solution here is. I guess we could ignore the legitimate/potentially valid status for zoology completely and just make all those names available? That sounds like the best approach to me if you all believe "potentially valid" is too confusing.

**Comment by yroskov** [comment]
>ignore the legitimate/potentially valid status for zoology completely and just make all those names available? 

As for me, it is a best solution. All nomenclature statuses should appear in the CoL in compliance with Code regulations, as (we assume) they present in the GSD.
